internal/app/app.go

```diff
@@ -37,7 +37,12 @@ func NewApp() *App {
 
 	h := handlers.New(s)
 
-	r := route.New(h)
+	deleteHanlder, err := handlers.NewDelete(s)
+	if err != nil {
+		logger.Log.Fatal(err.Error())
```

Comment:
 Стоит добавить ошибку, как параметр:
```suggestion
		logger.Log.Fatal("Fatal error occurred", zap.Error(err))
```